src/api/Intelligence.js

```diff
@@ -42,6 +47,32 @@ class Intelligence extends Base {
             },
         });
     }
+
+    /**
+     * Sends an AI request to supported LLMs and returns extracted key pairs and values.
+     *
+     * @param {AiExtractStructured} request - AI Extract Structured Request
+     * @return A successful response including the answer from the LLM.
+     */
+    async extractStructured(request: AiExtractStructured): Promise<AiExtractResponse> {
+        this.errorCode = ERROR_CODE_EXTRACT_STRUCTURED;
+
+        const url = `${this.getBaseApiUrl()}/ai/extract_structured`;
+
+        let suggestionsResponse = {};
+        try {
+            suggestionsResponse = await this.xhr.post({
+                url,
+                data: request,
+            });
+        } catch (e) {
+            const { status } = e;
+            if (isUserCorrectableError(status)) {
```

Comment:
What happens if it's not a user correctable error?

Reply:
The logic here is also present in getTemplates and getInstances. We're only going to re-throw the error if it can be fixed by the user (like authorization or rate limitting).

You can refer to similar discussion we had at my previous PR [feat: support metadata suggestions API](https://github.com/box/box-ui-elements/pull/3571#top)